vpdq/python/vpdq_util.py

```diff
@@ -0,0 +1,88 @@
+import vpdq
+import numpy as np
+
+
+def vector_to_hex(hash_vector):
```

Comment:
This is also too much of copy of `pdq_hasher.py`
maybe we should move more of this into python-threatexchange so these kind of utils can be reused.